connector/MongoDBAtlasODBC.pq

```diff
@@ -140,14 +159,15 @@ TransformColumnKeepType = (table, columnName, operation) =>
     in
         Value.ReplaceType(TransformColumn, TableType);
 
-GetConnectionString = (uri as text, database as text) as record =>
+GetConnectionString = (uri as text, database as text, loglevel as text) as record =>

```

Comment:
Now that the loglevel is optional, it can use a check for the input not being set.  Something like this should work:
```
GetConnectionString = (uri as text, database as text, loglevel as text) as record =>
    let
        loglevelVal = if loglevel = null then "" else "info",
        DatabaseNotSet = (database = ""),
        ConnectionString = [
            DRIVER = "MongoDB Atlas SQL ODBC Driver",
            DATABASE = database,
            URI = uri,
            APPNAME = "powerbi-connector",
            LOGLEVEL = loglevel
        ]
```

Reply:
I had done this up above in each datasource method, but have consolidated them to here.